src/components/grapher/Dot.vue

```diff
@@ -48,22 +48,41 @@ import Vue from "vue";
 import StuntSheet from "@/models/StuntSheet";
 import DotAppearance from "@/models/DotAppearance";
 
+const nextSSDotAppearance = new DotAppearance({
+  filled: true,
+  fill: "purple",
+  color: "purple",
+});
+
+const nextSSUnconnectedDotAppearance = new DotAppearance({
+  ...nextSSDotAppearance,
+  fill: "deeppink",
+  color: "deeppink",
+});
+
 /**
  * Renders a single dot
  */
 export default Vue.extend({
   name: "Dot",
   props: {
+    isNextSS: Boolean,
     dotTypeIndex: Number,
     label: String,
     labeled: Boolean,
     selected: Boolean,
+    isConnected: Boolean,
   },
   computed: {
     radius(): number {
-      return 0.7;
+      return this.$props.isNextSS ? 0.5 : 0.7;
     },
     dotAppearance(): DotAppearance {
+      if (this.$props.isNextSS) {
+        return this.$props.isConnected
+          ? nextSSDotAppearance
+          : nextSSUnconnectedDotAppearance;
+      }
```

Comment:
Maybe we should have the appearance of the connected dots mimic the starting dot's appearance? It might make situations like this where multiple different dot types have paths that overlap a bit clearer:
<img width="486" alt="image" src="https://user-images.githubusercontent.com/29964426/123566928-d505ba80-d775-11eb-8e83-f5d26f5d6d8e.png">

Reply:
Good idea... Let's re-visit this once we get this in the hands of STUNT.